scalameta/src/main/scala/scala/meta/internal/parsers/Parsers.scala

```diff
@@ -562,6 +562,18 @@ private[meta] class Parser(val input: Input)(implicit val dialect: Dialect) { pa
     }
   }
 
+  // corresponding quasiquote (rank 1) for expressions like `{..$smth}`
+  def constructR1Quasi[T <: Tree : AstMetadata](ctor: => Tree): List[T] = {
```

Comment:
I don't know how to pass `@` as a parameter, getting compile errors. So, with
`def ellipsis[T <: Tree : AstMetadata](rank: Int, body: => Tree, skip: Any = 42): T = autoPos { ...`
````annots += ellipsis(1, expr(), skip = `@`)```
,I get weird `not found: value <none>
[error]         annots += ellipsis(1, expr(), skip = 2)
[error]                           ^
`